dog-blog-page3.html

```diff
@@ -0,0 +1,76 @@
+          <h4><a href="dog-checkout-page4.html" class="checkbutton">Checkout</a><h4>
+        </div>
+      </header>
+      <nav>
+        <ul class="main-navigation">
+          <li><a href="index.html" class="topnav">Home</a></li>/
+          <li><a href="show-dogs-page2.html" class="topnav">Dogs</a></li>/
+          <li><a href="dog-checkout-page4.html" class="topnav">Adopt</a></li>
+        </ul>
+      </nav>
+      <main>
+         <div>
+           <h1>Adoptadog DogaBlog</h1>
+          </div>
+        <article class="main-blog-para">
+        <div class="intro-para">
+          <img src="images/blog-1.jpg" width="500" alt="a pet owner hugs his dog while both look at a grand desert vista">
+        </div>
+        <div>
+            <h2>The Best Ways to Keep Your Puppy Travel-happy</h2>
+               <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary. Roll Over kitty barky critters litter stick window litter box wagging field toy. Whiskers harness biscuit food lick small animals throw meow house train. Bedding field hamster small animals carrier polydactyl groom vaccine. Commands running gimme five groom slobber run fast head ball litter box biscuit catch run fast roll over. Roll Over litter box tabby pet slobbery play dead kitty roll over small animals barky good boy string kitty fish licks teeth chew drool. ID Tag barky lick parakeet wet nose ball walk tabby wag tail chirp nest. Spike bird seed water teeth leash ball. Crate pet food stripes carrier drool slobbery tabby dog puppy birds walk roll over bird food Mittens pet supplies.
+               </p>
+         </div>
+      </article>
+      <article class="main-blog-para">
+         <div  class="intro-para">
+           <img src="images/blog-2.jpg" width="500" alt="">
+        </div>
+        <div>
+             <h2>How to Walk Mulitple Dogs</h2>
+                <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary. Roll Over kitty barky critters litter stick window litter box wagging field toy. Whiskers harness biscuit food lick small animals throw meow house train. Bedding field hamster small animals carrier polydactyl groom vaccine. Commands running gimme five groom slobber run fast head ball litter box biscuit catch run fast roll over. Roll Over litter box tabby pet slobbery play dead kitty roll over small animals barky good boy string kitty fish licks teeth chew drool. ID Tag barky lick parakeet wet nose ball walk tabby wag tail chirp nest. Spike bird seed water teeth leash ball. Crate pet food stripes carrier drool slobbery tabby dog puppy birds walk roll over bird food Mittens pet supplies.
+                </p>
+          </div>
+      </article>
+      <section class="main-blog-para">
```

Comment:
make sure all of these blog posts follow the same structure. This section seems out of place. Since they are all the same they can share the same markup